CHANGELOG.md

```diff
@@ -4,13 +4,41 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Common Changelog](https://common-changelog.org/).
 
+[1.2.0]: https://github.com/sablier-labs/v2-core/compare/v1.1.2...v1.2.0
 [1.1.2]: https://github.com/sablier-labs/v2-core/compare/v1.1.1...v1.1.2
 [1.1.1]: https://github.com/sablier-labs/v2-core/compare/v1.1.0...v1.1.1
 [1.1.0]: https://github.com/sablier-labs/v2-core/compare/v1.0.2...v1.1.0
 [1.0.2]: https://github.com/sablier-labs/v2-core/compare/v1.0.1...v1.0.2
 [1.0.1]: https://github.com/sablier-labs/v2-core/compare/v1.0.0...v1.0.1
 [1.0.0]: https://github.com/sablier-labs/v2-core/releases/tag/v1.0.0
 
+## [1.2.0] - 2024-06-16
+
+### Changed
+
+- **Breaking:** Move common logic into `Lockup` contract ([#813](https://github.com/sablier-labs/v2-core/pull/813),
+  [#850](https://github.com/sablier-labs/v2-core/pull/850), [#784](https://github.com/sablier-labs/v2-core/pull/784))
+- Disallow zero `startTime` ([#852](https://github.com/sablier-labs/v2-core/pull/852),
+  [#941](https://github.com/sablier-labs/v2-core/pull/941))
+- Change permissions of `withdraw` function to public ([#785](https://github.com/sablier-labs/v2-core/pull/785))
+- Refactor functions name in `ISablierV2Recipient` hook ([#781](https://github.com/sablier-labs/v2-core/pull/781))
+- Upgrade to OpenZeppelin v5 ([#806](https://github.com/sablier-labs/v2-core/pull/806))
+- Upgrade build dependencies ([#944](https://github.com/sablier-labs/v2-core/pull/944),
```

Comment:
Let's say "Bump"

Reply:
Addressed in https://github.com/sablier-labs/v2-core/pull/863/commits/c5664afc161dc46eba904d5c96a2584cd9e446db.